pkg/trace/agent/agent_test.go

```diff
@@ -1006,3 +1007,96 @@ func tracesFromFile(file string) (raw []byte, count int, err error) {
 	}
 	return data, count, nil
 }
+
+func TestProcessStats(t *testing.T) {
+	testCases := []struct {
+		in            pb.ClientStatsPayload
+		lang          string
+		tracerVersion string
+		out           pb.StatsPayload
+	}{
+		{
+			in: pb.ClientStatsPayload{
```

Comment:
@piochelepiotr I added fields to help down stream detect redelivery properly (runtime id and sequence number) [here](https://github.com/DataDog/dd-trace-java/pull/2583) I would like these to be included in the v0.6 format. Thanks.

Reply:
Just added, does that look good @richardstartin ?

Reply:
Looks good. I will update to target v0.6 on Monday.